Sources/AppStoreConnectClient/Models/BuildsQuery.swift

```diff
@@ -27,7 +27,63 @@ import Foundation
 /// A structure representing a query for builds with specified sort and fields parameters.
 public struct BuildsQuery {
     /// The sort payload for the query.
-    let sort: Operations.builds_hyphen_get_collection.Input.Query.sortPayload
+    public let sort: [Sort]?
     /// The fields payload for the query.
-    let fields: Operations.builds_hyphen_get_collection.Input.Query.fields_lbrack_builds_rbrack_Payload
+    public let fields: [Fields]?
+    
+    /// Converts the Sort enum to the appropriate payload type used in the query.
+    /// - Parameter from: The Sort enum value to convert.
+    /// - Returns: The converted sort payload.
+    func convertSort(_ from: [Sort]?) -> [Operations.builds_hyphen_get_collection.Input.Query.sortPayloadPayload]? {
```

Comment:
`func convertSort(_ from: [Sort]?) -> [Operations.builds_hyphen_get_collection.Input.Query.sortPayloadPayload]?` it is so-so need something like this `func convertSort(from sorts: [Sort]) -> [Operations.builds_hyphen_get_collection.Input.Query.sortPayloadPayload]`

Reply:
Changed.